src/main/java/mate/academy/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.ShoppingCartDao;
+import mate.academy.dao.TicketDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.MovieSession;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.Ticket;
+import mate.academy.model.User;
+import mate.academy.service.ShoppingCartService;
+import mate.academy.service.UserService;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+    private TicketDao ticketDao;
+    private UserService userService;
```

Comment:
```suggestion
    @Inject
    private TicketDao ticketDao;
    @Inject
    private UserService userService;
```

Did you test your solution? Does it work without @Inject annotation? Do you use userService in this class?